express_relay/sdk/js/src/index.ts

```diff
@@ -0,0 +1,285 @@
+import type { paths } from "./types";
+import createClient, { ClientOptions } from "openapi-fetch";
+import {
+  Address,
+  encodeAbiParameters,
+  encodePacked,
+  Hex,
+  isAddress,
+  isHex,
+  keccak256,
+} from "viem";
+import { privateKeyToAccount, sign, signatureToHex } from "viem/accounts";
+
+/**
+ * ERC20 token with contract address and amount
+ */
+export type TokenQty = {
+  contract: Address;
+  amount: bigint;
+};
+
+/**
+ * Bid information
+ */
+export type BidInfo = {
+  /**
+   * Bid amount in wei
+   */
+  amount: bigint;
+  /**
+   * Unix timestamp for when the bid is no longer valid in seconds
+   */
+  valid_until: bigint;
+};
+
+/**
+ * All the parameters necessary to represent a liquidation opportunity
+ */
+export type OpportunityParams = {
```

Comment:
imo `Opportunity` would be a better name here. The current name sounds like the configuration parameters that you use to filter opportunities or something like that.

Reply:
The original rational here was to keep the names similar to the openapi schema names which reflects the backend structs. But I agree that `Opportunity` is cleaner and users shouldn't see all the details we see in the backend :D